lightning/src/onion_message/messenger.rs

```diff
@@ -795,8 +795,8 @@ pub trait CustomOnionMessageHandler {
 pub enum PeeledOnion<T: OnionMessageContents> {
 	/// Forwarded onion, with the next node id and a new onion
 	Forward(NextMessageHop, OnionMessage),
-	/// Received onion message, with decrypted contents, path_id, and reply path
-	Receive(ParsedOnionMessageContents<T>, Option<[u8; 32]>, Option<BlindedPath>)
+	/// Received onion message, with decrypted contents, recipient data, and reply path
```

Comment:
s/recipient data/context

Reply:
Done! [528dc64](https://github.com/lightningdevkit/rust-lightning/commit/528dc644e5b995c10f96fe36fb803bc6433580d7)
Thanks for the pointer!